lisp/mastodon-toot.el

```diff
@@ -69,10 +69,19 @@ Set `mastodon-toot--content-warning' to nil."
   (interactive)
   (mastodon-toot--kill))
 
+(defun mastodon-toot--remove-docs ()
+  "Get the body of a toot from the current compose buffer."
+  (let ((re "^|=+=|$"))
```

Comment:
I haven't cherry-picked and tried this out yet, but I'm surprised you don't need to escape those pipes.

```elisp
"^\|=+=\|$"
```